### PR TITLE
Git Sync/dbt Extension Updates

### PR DESCRIPTION
**Pull request by pwseg**
### Proposed changes

- Added troubleshooting sections to both Git Sync and dbt Extensions pages.
- Added section on Git Connections.
- Some other minor cleanup.

### Merge timing

- ASAP once approved.

### CODE REVIEW

File: src/segment-app/extensions/dbt.md [hunk 1]
```diff
@@ -69,3 +69,44 @@ After you've successfully set up dbt with a warehouse and connected to your Git
 6. Enter your **Model Name**, then click **Create Model**.
 
 To change a connected model, ensure that you've removed it from all active Reverse ETL syncs.
+
+## Git Connections
+
+Git Connections enable Segment to sync data with your preferred Git repository through supported connection types. Segment supports several connection options connecting, including SSH and token-based authentication.
```
Comment (forstisabella) [inline src/segment-app/extensions/dbt.md]:
```suggestion
Git Connections enable Segment to sync data with your preferred Git repository through supported connection types. Segment supports several connection options, including SSH and token-based authentication.
```

File: src/segment-app/extensions/dbt.md [hunk 2]
```diff
@@ -69,3 +69,44 @@ After you've successfully set up dbt with a warehouse and connected to your Git
...
+
+> info ""
+> Git Sync and the dbt integration operate independently. You don’t need Git Sync set up to use dbt, and dbt Cloud can trigger its own syncs without relying on Git Sync.
```
Comment (forstisabella) [inline src/segment-app/extensions/dbt.md:78]:
```suggestion
> Git Sync and the dbt integration operate independently. You don’t need to set up Git Sync to use dbt, and dbt Cloud can trigger its own syncs without relying on Git Sync.
```

File: src/segment-app/extensions/dbt.md [hunk 3]
```diff
@@ -69,3 +69,44 @@ After you've successfully set up dbt with a warehouse and connected to your Git
...
+
+### Supported connection types
+
+Segment supports the following credential types for setting up a Git Connection:
+
+- **SSH**: Compatible with GitHub, GitLab, and Bitbucket, SSH provides a secure method for connecting to your repository.
+- **Git Token**: Git tokens are also supported across GitHub, GitLab, and Bitbucket, enabling token-based authentication for added flexibility.
```
Comment (forstisabella) [inline src/segment-app/extensions/dbt.md:85]:
```suggestion
- **Git token**: Git tokens are supported across GitHub, GitLab, and Bitbucket, enabling token-based authentication for added flexibility.
```

File: src/segment-app/extensions/dbt.md [hunk 4]
```diff
@@ -69,3 +69,44 @@ After you've successfully set up dbt with a warehouse and connected to your Git
...
+
+### Reusing Git Connections
+
+Segment lets you set up multiple Git Connections, allowing you to reuse credentials across both dbt and Git Sync. You can either use the same credential for multiple configurations or create separate Git Connections for each product and environment as needed.
+
+If you plan to reuse a Git token across both dbt and Git Sync, ensure it has the necessary read and write permissions for both integrations.
+
+## Setting Up CI checks
+
+> info "CI Check availability"
```
Comment (forstisabella) [inline src/segment-app/extensions/dbt.md:96]: If "CI Check" is a proper noun then don't worry about this suggestion

```suggestion
> info "CI check availability"
```

File: src/segment-app/extensions/dbt.md [hunk 5]
```diff
@@ -69,3 +69,44 @@ After you've successfully set up dbt with a warehouse and connected to your Git
...
+> CI Checks are available only with the GitHub App connection.
```
Comment (forstisabella) [inline src/segment-app/extensions/dbt.md:97]:
```suggestion
> CI checks are available only with the GitHub App connection.
```

File: src/segment-app/extensions/git.md [hunk 6]
```diff
@@ -76,3 +83,34 @@ To manage Segment resources using Git and Terraform, follow these steps:
 
 
 For more information on using Terraform, visit [Terraform's documentation](https://developer.hashicorp.com/terraform/docs){:target="_blank"}.
+
+## Git Connections
+
+Git Connections enable Segment to sync data with your preferred Git repository through supported connection types. Segment supports several connection options connecting, including SSH and token-based authentication.
```
Comment (forstisabella) [inline src/segment-app/extensions/git.md]:
```suggestion
Git Connections enable Segment to sync data with your preferred Git repository through supported connection types. Segment supports several connection options, including SSH and token-based authentication.
```

File: src/segment-app/extensions/git.md [hunk 7]
```diff
@@ -76,3 +83,34 @@ To manage Segment resources using Git and Terraform, follow these steps:
...
+
+> info ""
+> Git Sync and the dbt integration operate independently. You don’t need Git Sync set up to use dbt, and dbt Cloud can trigger its own syncs without relying on Git Sync.
```
Comment (forstisabella) [inline src/segment-app/extensions/git.md:92]:
```suggestion
> Git Sync and the dbt integration operate independently. You don’t need to set up Git Sync to use dbt, and dbt Cloud can trigger its own syncs without relying on Git Sync.
```

File: src/segment-app/extensions/git.md [hunk 8]
```diff
@@ -76,3 +83,34 @@ To manage Segment resources using Git and Terraform, follow these steps:
...
+
+### Supported connection types
+
+Segment supports the following credential types for setting up a Git Connection:
+
+- **SSH**: Compatible with GitHub, GitLab, and Bitbucket, SSH provides a secure method for connecting to your repository.
+- **Git Token**: Git tokens are also supported across GitHub, GitLab, and Bitbucket, enabling token-based authentication for added flexibility.
```
Comment (forstisabella) [inline src/segment-app/extensions/git.md]:
```suggestion
- **Git token**: Git tokens are supported across GitHub, GitLab, and Bitbucket, enabling token-based authentication for added flexibility.
```

File: src/segment-app/extensions/git.md (hunk 6 above)
Comment (pwseg) [reply src/segment-app/extensions/git.md]: I caught both of these myself when I was reviewing!  Looks like they've already been updated.

**Review by forstisabella — APPROVED** [review]

**Review by forstisabella — APPROVED** [review]

**Pull request merged**